apps/glusterfs/allocator_simple.go

```diff
@@ -10,8 +10,9 @@
 package glusterfs
 
 import (
-	"github.com/boltdb/bolt"
 	"sync"
+
+	"github.com/boltdb/bolt"
```

Comment:
@raghavendra-talur if there is a case for this change -- make a separate patch (and maybe separate PR).